images/libvirt-kubevirt/Dockerfile

```diff
@@ -44,4 +44,12 @@ RUN rm -f /libvirtd.sh
 COPY libvirtd.sh /libvirtd.sh
 RUN chmod a+x /libvirtd.sh
 
+RUN echo "unix_sock_group=\"libvirt\"" >> /etc/libvirt/libvirtd.conf
+RUN echo "unix_sock_ro_perms=\"0777\"" >> /etc/libvirt/libvirtd.conf
+RUN echo "unix_sock_rw_perms=\"0770\"" >> /etc/libvirt/libvirtd.conf
+RUN echo "auth_unix_ro=\"none\"" >> /etc/libvirt/libvirtd.conf
+RUN echo "auth_unix_rw=\"none\"" >> /etc/libvirt/libvirtd.conf
```

Comment:
Thanks for comments guys, I checked the possibility to change auth to SASL and it will work, but we will need to explicitly send the same user and password that we used for `saslpasswd2` command under calls to `virtwrap.NewConnection`.
I believe we can define some default values for user and password under `config.sh` and use it when we call to the container, something like
```
command:
          - "/virt-handler"
          - "-v"
          - "3"
          - "--libvirt-uri"
          - "qemu:///system"
          - "--hostname-override"
          - "$(NODE_NAME)"
          - {{ sasl_user }}
          - {{ sasl_password }}
```
@rmohr @berrange @fabiand Give me know if it looks fine for you.